pkgs/tools/networking/imapsync/default.nix

```diff
@@ -19,18 +27,44 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  buildInputs = with perlPackages; [ perl openssl MailIMAPClient TermReadKey
-    IOSocketSSL DigestHMAC URI FileCopyRecursive IOTee UnicodeString
-    DataUniqid JSONWebToken TestMockGuard LWP CryptOpenSSLRSA
-    LWPProtocolHttps Readonly TestPod TestMockObject ParseRecDescent
-    IOSocketInet6 NTLM
+  buildInputs = with perlPackages; [
+    Appcpanminus
+    CGI
+    CryptOpenSSLRSA
+    DataUniqid
+    DistCheckConflicts
+    EncodeIMAPUTF7
+    FileCopyRecursive
+    FileTail
+    IOSocketInet6
+    IOTee
+    JSONWebToken
+    LWPUserAgent
+    MailIMAPClient
+    ModuleImplementation
+    ModuleScanDeps
+    NTLM
+    PARPacker
+    PackageStash
+    PackageStashXS
+    Readonly
+    RegexpCommon
+    SysMemInfo
+    TermReadKey
+    TestDeep
+    TestFatal
+    TestMockGuard
+    TestMockObject
+    TestPod
+    TestRequires
+    UnicodeString
+    perl
   ];
 
   meta = with lib; {
-    homepage = "http://www.linux-france.org/prj/imapsync/";
+    homepage = "https://imapsync.lamiral.info/";
     description = "Mail folder synchronizer between IMAP servers";
-    license = licenses.gpl2Plus;
-    platforms = platforms.linux;
```

Comment:
Please set platforms again.

Reply:
imapsync supports Darwin (see https://imapsync.lamiral.info/INSTALL.d/INSTALL.Darwin.txt), that's why I've remove that line. It's unfortunately currently blocked by perlPackages.CryptOpenSSLRandom being broken on Darwin (seems similar to python3Packages.pyopenssl to me). Should we add `broken = stdenv.isDarwin;`?

Reply:
Then we should add ``platforms = platforms.unix;`` and yes please mark ``perlPackages.CryptOpenSSLRandom`` broken on darwin

Reply:
Thanks, done. perlPackages.CryptOpenSSLRandom is already marked as broken on Darwin.